features/stamp_duty_information_table.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Stamp Duty Information Table
+In order to understand how my stamp duty is calculated
+As a home buyer
+I want to see what percentages apply to the purchase price
+
+Scenario: As a home buyer
+  Given I visit the Stamp Duty page
+  And I select 'isFTB' as my buyer type
```

Comment:
`And I select 'isFTB' as my buyer type` --> `When I am buying a home for the first time`.

The reason for this suggestion is that the implementation behind the scenes does not need to be part of story-telling. This is just a suggestion though.